apollo-federation/src/sources/connect/json_selection/methods.rs

```diff
@@ -147,6 +148,7 @@ impl std::ops::Deref for ArrowMethod {
             Self::Size => &public::SizeMethod,
             Self::Entries => &public::EntriesMethod,
             Self::JsonStringify => &public::JsonStringifyMethod,
+            Self::Join => &public::JoinMethod,
```

Comment:
Please move this to the `::future` section until we've ironed out all the policy decisions involved in this method's design. There are several other future methods I feel better about than `->join`, so I don't want it skipping the line. It needs time to bake.

Reply:
i'm not going to merge it until it's ready, but it will be public